dist/estrella.js

```diff
@@ -30,7 +30,7 @@ Example of using estrella with a build script:
+  `},mt=Promise.resolve(),M={},X=[],De=Symbol("IS_MAIN_CALL");var Or=!1;function Br(e){if(!Or){Or=!0;let t=e||1;process.exitCode=t,process.on("exit",r=>{process.exit(r||t)})}}function vn(e){if(a.debug(()=>`input config ${(0,y.repr)(e)}`),e.entryPoints||(e.entryPoints=[]),e.entry&&(Array.isArray(e.entry)?e.entryPoints=e.entryPoints.concat(e.entry):e.entryPoints.push(e.entry)),delete e.entry,e.entryPoints.length==0&&!e.stdin&&(a.debug(()=>"missing entryPoints; attempting inference"),e.entryPoints=vr(e),e.entryPoints.length==0)){let t=Ce(e)?" (could not guess from tsconfig.json)":"";throw new ce(`config.entryPoints is empty or not set${t}`)}if(e.sourcemap?e.sourcemap!="inline"&&e.sourcemap!="external"&&(e.sourcemap=!0):e.sourcemap=!1,!e.outfile&&!e.outdir){e.setOutfile("-");let t=e.updateProjectID();e.setOutfile(Ee.join((0,y.tmpdir)(),`esbuild.${t}.out.js`)),e.outfileIsTemporary=!0}e.updateProjectID(),a.debug(()=>`effective config for project#${e.projectID}: ${(0,y.repr)(e)}`)}function jn(e,t){let r=(0,y.clock)(),n=JSON.parse(oe.readFileSync(e));for(let o in t){let i=t[o];i===void 0?delete n[o]:(typeof i=="function"&&(i=i(n[o])),n[o]=i)}oe.writeFileSync(e,JSON.stringify(n)),a.debug(()=>`patched source map ${e} with overrides ${(0,y.repr)(t)} (${(0,y.fmtDuration)((0,y.clock)()-r)})`)}function vr(e){let t=Ce(e);if(t){if(a.debug(()=>`tsconfig file found at ${he(e)}`),t.files)return t.files;if(t.include){let r=[];for(let n of t.include)a.debug(`guessing entry points: glob.glob(${n}) =>`,fe.glob(n)),r=r.concat(fe.glob(n));if(t.exclude)for(let n of t.exclude)r=r.filter(o=>!fe.match(n,o));return r.slice(0,1)}}return[]}function An(e){let t={},r={},n={name:"globalName"};for(let o of Object.keys(e))Tr.BuildConfig.has(o)||(_r.BuildOptions.has(o)||(r[o]=e[o]),o=n[o]||o,t[o]=e[o]);return Object.keys(r).length>0&&a.info(`Notice: Potentially invalid esbuild.BuildOption(s): ${(0,y.repr)(r)}
```

Comment:
Please revert and remove changes to this file

Reply:
@rsms Sorry. When I was writing the test it wouldn't pass unless I compiled into dist. Maybe I'm running in wrong, but right now anytime I run the tests is recompiles dist.

Reply:
You're doing it right :-) Just don't check in the changes to files in the dist folder. (When you do `git add` don't include files in dist)

Reply:
BTW, I'm considering just removing the dist directory from version control entirely and relying on people using npm for releases. Would make it possible to simply do `git add .`